.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,17 @@
-This pull request is related to issue # <!-- insert the issue number behind the # or delete the line -->
+<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->
+Fixes #
 
 I modified the labyrinth, you can test my changes here:
-http://rawgit.com/USERNAME/labyrinth/BRANCH-NAME/index.html
-<!-- Please replace USERNAME with your GitHub user name and
-     BRANCH-NAME with your branch name e.g. master -->
+ http://rawgit.com/USERNAME/labyrinth/BRANCH-NAME/index.html
+ <!-- Please replace USERNAME with your GitHub user name and
+      BRANCH-NAME with your branch name e.g. master -->
+#### Checklist
 
+- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
+- [ ] My branch is up-to-date with the Upstream `master` branch.
+- [ ] I have added necessary documentation (if appropriate)
 
-<!-- please summarize the problem you faced -->
-The problem I want to solve is 
```

Comment:
Having a problem described apart from the solution is a good thing.
This should be the task of the issue, too. If the issue is bigger than on e pull request, this can help understand which part of the issue is solved.

One Problem, many solutions.

Just describing changes could lead to something like "add call to function XY in function Z" but we need something like: The problem I faced was ... There were these solutions: 1, 2, 3 and I decided to use 3 because ...